compiler/arser/tests/arser.test.cpp

```diff
@@ -206,3 +206,26 @@ TEST(BasicTest, MultipleFloatValue)
 
   EXPECT_THROW(arser.get<std::vector<int>>("--add_float"), std::runtime_error);
 }
+
+TEST(BasicTest, MultipleStringValue)
+{
+  /* arrange */
+  Arser arser;
+
+  arser.add_argument("--three_color")
+      .nargs(3)
+      .type(arser::DataType::STR_VEC)
```

Comment:
(Just curiosity) What if I do something like this?

```cpp
      .nargs(3)
      .type(arser::DataType::STR)
```

Reply:
@wateret Then, it will take three inputs but user won't be able to "get" those values. `arser.get<std::string>` gives an error with type mismatch